pages/k8s/how-to-cos-lite.md

```diff
@@ -0,0 +1,138 @@
+---
+wrapper_template: "templates/docs/markdown.html"
+markdown_includes:
+  nav: "kubernetes/docs/shared/_side-navigation.md"
+context:
+  title: "Integrating COS Lite with Charmed Kubernetes"
+  description: Integrating COS Lite with Charmed Kubernetes
+keywords: Cloud, cluster, observability
+tags: [observability, operating]
+sidebar: k8smain-sidebar
+permalink: how-to-cos-lite.html
+layout: [base, ubuntu-com]
+toc: False
+---
+
+First, initialize a controller on your preferred cloud platform (vSphere, AWS,
+Azure) using Juju. Run the following command, ensuring to replace placeholder
+values with your specifics:
+
+```
+juju bootstrap --credential $XYZ \
+  --debug \
+  --model-default automatically-retry-hooks=false \
+  --model-default 'logging-config=<root>=DEBUG' \
+  --bootstrap-image="juju-ci-root/templates/jammy-test-template" \
+  --bootstrap-series=jammy \
+  --bootstrap-constraints "arch=amd64" \
+  aws tutorial-controller
+```
+
+Next, create a microk8s model to act as a deployment cloud for cos-lite:
+
+```
+juju add-model --config logging-config='<root>=DEBUG' \
+  microk8s-ubuntu aws
+```
+
+Deploy Ubuntu on the microk8s-ubuntu model with the following specifications:
+
+```
+juju deploy ubuntu --series=focal --constraints="mem=16G cores=8 root-disk=50G"
```

Comment:
I would suggest it is more specific than that, because it gets confusing when you are referring to `microk8s`  itself. Could we call it something like 'microk8s-node' or 'o11y-k8s` or something less confusing